egs2/asvspoof/spk1/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+set -u
+set -o pipefail
+
+
+spk_config=conf/train_rawnet3.yaml
+
+train_set="train"
+valid_set="test"
+cohort_set="test"
```

Comment:
Please note that the cohort set cannot be equal to the test set. 
Could you use the development set since ASVspoof2019 LA explicitly has this partition?

Reply:
I see. Thanks for making me aware of this. I have added support for the dev partition, and I've assigned it to be the valid set and cohort set